Source/SwiftLintFramework/Rules/CommaRule.swift

```diff
@@ -13,7 +13,7 @@ public struct CommaRule: CorrectableRule {
     public static let description = RuleDescription(
         identifier: "comma",
         name: "Comma Spacing",
-        description: "One space before and no after must be present next to any comma.",
+        description: "One space after and no before must be present next to any comma.",
```

Comment:
If we're discussing wording, this should probably say "No space before and one after...".